mobileapps/views.py

```diff
@@ -296,3 +300,206 @@ def delete(self, request, mobileapp_id):
         except ObjectDoesNotExist:
             raise Http404
 
+
+class MobileAppAllUsersNotifications(SecureListCreateAPIView):
```

Comment:
I think these APIs should inherit from `SecureCreateAPIView` since we are not listing anything here.